in_toto/in_toto_record.py

```diff
@@ -156,9 +166,11 @@ def main():
 
   args = parser.parse_args()
 
-  # Turn on all the `log.info()` in the library
-  if args.verbose:
-    log.logging.getLogger().setLevel(log.logging.INFO)
+  # Distinguish log levels via parameterized input
+  log.logging.getLogger().setLevel(args.loglevel)
+
+  # Set color settings
+  in_toto.settings.COLOR = args.colorize
```

Comment:
The problem with this approach is that it ignores the defaults in `in_toto.settings`. Can you change both to only override settings if the options are passed?

Reply:
I've set the `default` parameter from the `argsparse` library to whatever is specified in `LOG_LEVEL` (from the settings module). Is that good enough? 

I also have an additional check afterwards, but I think it might be unnecessary because the default specified in `argsparse` might do the job. If you could look over that, i'd appreciate it.

Reply:
If you don't mind just wrap it in an if.

Reply:
I was asking because the coveralls API seems to drop coverage when certain statements are written (I don't know if this was intentional or not).

Additionally: are unit tests required for what's in the log module? The reason I ask is because it seems to be saying the debug function in the log module is missing in terms of "coverage".